index.html

```diff
@@ -335,5 +240,6 @@ <h2 class="intro">
         </li>
       </ul>
     </footer>
+    <script type="module" src="./js/script.js"></script>
```

Comment:
Thank you for implementing this change, as requested.  

_About the live demo, I'm taking this as OPTIONAL, because a deployment was not a requirement for this particular project. Since all the other requested changes were implemented, I'm approving this project._